### PR TITLE
Ahackett/listscroll525

### PR DESCRIPTION
**Pull request by ahackettms**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes

List scrolling example is not working well. 
Without autobind, this.state is null in some of the methods

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx [hunk 1]
```diff
@@ -69,6 +70,7 @@ export class ListScrollingExample extends React.Component<IListScrollingExampleP
           <TextField
             value={ this.state.selectedIndex.toString(10) }
             onChanged={ this._onChangeText }
+            disabled={ true }
```
[the remark this answers is not in the document]
Comment (cschleiden) [reply packages/office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx]: Why can't you enter a value? It looks broken on the website right now, but it used to work.

[the remark this answers is not in the document]
Comment (ahackettms) [reply packages/office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx]: I don't know the code history about this example page, I just saw it not working after I select item from the dropdown or click on the -1, -10, +1, +10, etc and fixing it with the @autobind
I don't know whether initially user supposedly be able to enter the value or not. But it seems to me that that is just binding to the selectedIndex value and selectedIndex is changed by the dropdown change or click on (-1, -10, +1, +10).

[the remark this answers is not in the document]
Comment (cschleiden) [reply packages/office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx]: Can you just remove the disabled?

[the remark this answers is not in the document]
Comment (ahackettms) [reply packages/office-ui-fabric-react/src/components/List/examples/List.Scrolling.Example.tsx]: done

**Pull request merged**